tests/ios/frameworks/strict-deps/a/BUILD.bazel

```diff
@@ -1,10 +1,29 @@
 load("//rules:framework.bzl", "apple_framework")
+load("//rules:substitute_build_settings.bzl", "substitute_build_settings")
 
 apple_framework(
     name = "a",
     srcs = glob(["*.swift"]),
     infoplists = ["//rules/test_host_app:Info.plist"],
     platforms = {"ios": "10.0"},
+    skip_packaging = ["plist"],
     visibility = ["//visibility:public"],
     deps = ["//tests/ios/frameworks/strict-deps/b"],
 )
+
+apple_framework(
```

Comment:
Thanks for adding a test on this - this was totally off peoples radar I think ❤️